src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+* Consider using a [single textarea for address](#address-textarea).
+* Use [autocomplete for billing address](#billing-address).
+* [Internationalize and localize](#internationalization-localization) where necessary.
+* Consider avoiding [postal code address lookup](#postal-code-address-lookup).
+* Use [appropriate payment card autocomplete values](#payment-form-autocomplete).
+* Use a [single input for payment card numbers](#single-number-input).
+* [Avoid using custom elements](#avoid-custom-elements) if they break the autofill experience.
+* [Test in the field as well as the lab](#analytics-rum): page analytics, interaction analytics, and 
+real-user performance measurement.
+* [Test on a range of browsers, devices and platforms](#test-platforms).
+
+{% Aside %}
+This article is about frontend best practices for address and payment forms. It does not explain 
+how to implement transactions on your site. To find out more adding payment functionality to your 
+website on the web, see [Web Payments](/payments).
+{% endAside %}
+
+## Use meaningful HTML {: #meaningful-html}
+
+Use the elements and attributes built for the job: 
+* `<form>`, `<input>`, `<label>`, and `<button>`
+* `type`, `autocomplete` and `inputmode`
+
+These enable built-in browser functionality, improve accessibility, and add meaning to your markup.
+
+### Use HTML elements as intended {: #html-elements}
+
+#### Put your form in a &lt;form&gt; {: #html-form}
+
+You might be tempted not to bother wrapping your `<input>`s in a `<form>`, and to handle data 
+  submission purely with JavaScript. 
+
+Don't do it!
+
+An HTML `<form>` gives you access to a powerful set of built-in features across all modern browsers, 
+  and can help make your site accessible to screen readers and other assistive devices. A `<form>` 
+  also makes it simpler to build basic functionality for older browsers with limited JavaScript 
+  support, and to enable form submission even if there's a glitch with your code—and for the 
+  small number of users who actually disable JavaScript.
+
+If you have more than one page component for user input, make sure to put each in its own `<form>` 
+  element. For example, if you have search and sign-up on the same page, put each in its own `<form>`.
+
+#### Use `<label>` to label elements {: #html-label}
+
+To label an `<input>`, `<select>` or `<textarea>`, use a [`<label>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label).
+
+You associate a label with an input by giving the label's `for` attribute the same value as 
+the input's `id`.
+
+```html
+<label for="address-line1">Address line 1</label>
+<input id="address-line1" …>
+```
+Use a single label for a single input: don't try to label multiple inputs with only one label. This 
+works best for browsers, and best for screenreaders. A tap or click on a label moves focus to the input 
+it's associated with, and screenreaders announce label text when the *label* or the label's *input* 
+gets focus.
```

Comment:
```suggestion
To label an `<input>`, `<select>`, or `<textarea>`, use a [`<label>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label).

Associate a label with an input by giving the label's `for` attribute the same value as 
the input's `id`.

```html
<label for="address-line1">Address line 1</label>
<input id="address-line1" …>
```
Use a single label for a single input: don't try to label multiple inputs with only one label. This 
works best for browsers, and best for screenreaders. A tap or click on a label moves focus to the input 
it's associated with, and screenreaders announce label text when the *label* or the label's *input* 
gets focus.
```